extensions/amp-list/validator-amp-list.protoascii

```diff
@@ -46,6 +46,7 @@ tags: {  # <amp-list>
   attrs: { name: "items" }
   attrs: { name: "max-items" }
   attrs: { name: "single-item" }
+  attrs: { name: "reset-on-refresh" }
```

Comment:
please add `value=""` that would ensure `reset-on-refresh` can not be given a value and is treated as a binary flag.
also alphabetical ordering of the attributes names